src/status_im/contexts/wallet/networks/validation.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im.contexts.wallet.networks.validation
+  (:require [malli.core]
+            [malli.error]))
+
+(def ?network
+  [:map-of
+   int?
+   [:map {:closed true}
+    [:network-name keyword?]
+    [:source :schema.common/image-source]
+    [:abbreviated-name string?]
+    [:block-explorer-name string?]]])
+
+(defn validate-network
```

Comment:
Is this a validation for network structure that should be universal across the app?
If yes - should we use it in data-store after turning the `status-go` network structure into ours?